python/scripts/consul/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -x
+set -e
+
+# generate proto files
+python -m grpc_tools.protoc -I ../../../protos/ --python_out=. --grpc_python_out . ../../../protos/metaspore.proto
+
+docker build -t consul-watch-load:v1.0.0 .
```

Comment:
Generate an absolute path for docker build context dir, e.g.
docker build -t consul-watch-load:v1.0.0 $(dirname "$0")

Reply:
> Generate an absolute path for docker build context dir, e.g. docker build -t consul-watch-load:v1.0.0 $(dirname "$0")

And also specify docker file absolute path with -f